src/containers/BooksForm.js

```diff
@@ -53,7 +54,7 @@ const BooksForm = () => {
             ))}
           </select>
         </label>
-        <button type="submit" className={classes.submit_btn}>Submit</button>
+        <button type="submit">ADD BOOK</button>
       </form>
     </section>
   );
```

Comment:
- [ ] Looking at the `Add New Book` section of the page, I think you can improve on the styling also. The stylings, positions, and spacings can be improved.
_**Original Page**_
![new](https://user-images.githubusercontent.com/26856491/119778261-34468580-bebf-11eb-9a05-83aac2d30133.png)
_**Clone Page**_
![pick](https://user-images.githubusercontent.com/26856491/119778281-390b3980-bebf-11eb-8038-643ac73e025a.png)